drake/systems/plants/RigidBodyTree.cpp

```diff
@@ -1326,6 +1326,31 @@ shared_ptr<RigidBody> RigidBodyTree::findLink(std::string linkname, int robot) c
   return nullptr;
 }
 
+shared_ptr<RigidBodyFrame> RigidBodyTree::findFrame(std::string frame_name, int robot) const
```

Comment:
note that i *really* want to move away from the int robotnum in c++.  i've got a version of findLink which takes in the robot string name now, and think we should be using that instead.